assets/js/state/selectors/index.js

```diff
@@ -10,6 +13,11 @@ export const isIdByKey =
 export const getHost = (id) => (state) =>
   state.hostsList.hosts.find((host) => host.id === id);
 
+export const getHostSelectedChecks = (hostID) => (state) => {
+  const host = getHost(hostID)(state);
+  return get(host, 'selected_checks', defaultEmptyArray);
+};
+
```

Comment:
Could you create a `host.js` file and move this inside? This way we could start moving all the host-related selectors into their own yard

Reply:
Yeah good idea! i moved only host related functions to `host.js`